home.spec.js

```diff
@@ -0,0 +1,30 @@
+var homePage = require('./pages/home');
+
+describe('Mercury Tours demo site', function () {
+
+	beforeEach(function () {
+		homePage.get();
+	});
+
+	it('hould have home page with title Welcome: Mercury Tours', function () {
+		expect(homePage.getTitle()).toEqual("Welcome: Mercury Tours");
+	});
+
+	it('should have footer with © 2005, Mercury Interactive (v. 011003-1.01-058)', function () {
+		expect(homePage.getText()).toEqual("© 2005, Mercury Interactive (v. 011003-1.01-058)");
```

Comment:
I think that getText is not enough meaningful name for the function. Please change it.